filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -937,8 +937,36 @@ void OpenGLDriver::framebufferTexture(backend::TargetBufferInfo const& binfo,
         attachmentTypeNotSupportedByMSRTT = true;
     }
 
-    if (rt->gl.samples <= 1 ||
-        (rt->gl.samples > 1 && t->samples > 1 && gl.features.multisample_texture)) {
+    // The following 4 techniques are different mechanisms to provide storage to the FBO.
+    // We'll use the first technique that is sufficient.
+    // nonMSAA:         MSAA was not requested, or we're rendering to an MS texture which
+    //                  doesn't require any special setup here
+    // useMSRTT:        use the EXT_multisampled_render_to_texture extension
+    // renderDirect:    texture is not sampleable, render directly into a renderbuffer
+    // emulateMSRTT:    prior 3 techniques are unsuitable, emulate MSRTT extension support
```

Comment:
Well, it's not that the 3 prior techniques are unsuitable -- it's more that we need an automatic resolve and that it can't be done by MSRTT.
And we need an automatic resolve because:
- the attachment **is not** MS
- the RenderTarget **is** MS

Which is our definition of "auto resolve" in our API.